packages/core/src/node/backend-application-module.ts

```diff
@@ -31,16 +30,7 @@ export const backendApplicationModule = new ContainerModule(bind => {
 
     bindServerProcess(bind, clusterRemoteMasterProcessFactory);
 
-    bind(DispatchingMessageClient).toSelf().inSingletonScope();
-    bind(MessageClient).toDynamicValue(ctx => ctx.container.get(DispatchingMessageClient)).inSingletonScope();
```

Comment:
On the frontend there is still a binding connecting to some websocket. If you remove this you also should remove that : https://github.com/theia-ide/theia/pull/932/files#diff-c7c547eb3348b187bd3f09089929bbf6R79